fibonacci.rb

```diff
@@ -0,0 +1,21 @@
+def fibonacci(current_num, cache)
+  if current_num === 0 || current_num === 1
+    1

```

Comment:
This is true. The way I solved it was with some janky preloading (since 0,1,1 or 1,1 will always be the first 3 or 2 elements) BUT that is tying particular knowledge to the code that wouldn't hold should the formula ever changed.

Thanks! Will fix it up.